### PR TITLE
feat: desktop download links/info

### PR DESCRIPTION
**Pull request by Rexogamer**
This commit also adds the developers link to the navbar.

## Screenshots
image
image
image
image

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -50,6 +51,7 @@ export default function Home() {
                                 <p>Don't bother with other chat apps.</p>
                                 <h1>Communication is critical.<br />Privacy is essential.</h1>
                                 <JoinLinks />
+                                <h3>(<Link href="/desktop">See here</Link> for more information on downloading Revolt Desktop.)</h3>
```
Comment (RigidStudios) [inline pages/index.tsx:54]:
```suggestion
                                <h3><Link href="/desktop">See here</Link> for more information on downloading Revolt Desktop.</h3>
```
Remove the parentheses, text looks awkward, you should probably even consider finding a different color or text size for it.

Comment (Rexogamer) [reply pages/index.tsx:54]: I feel like it looks a bit awkward *without* the parentheses, but I'd like to hear a few more thoughts. I'll try and colour the link.

**Comment by insertish** [comment]
I feel like it clutters up the home page too much, it's fine in footer but probably shouldn't go on main page until we're out of beta.

The promo code has limited uses so I may release it on store first before including it here.

Either way the installation flow could definitely be a lot nicer:
- Steam has a very simple way to lead people to the correct file: https://store.steampowered.com/about/
- Paint.NET offers both options (on Windows) as two big buttons: https://www.getpaint.net/download.html
  I would probably have MS Store side by side with installer on something like the front page.

**Comment by Rexogamer** [comment]
Fine with waiting until we're out of beta, some notes:
- Steam hosts their files using Cloudflare, while PDN just pushes them to a subdirectory - if the release .exes were automatically pushed to a server (eg `download.revolt.chat`) that would work fine
- A MS Store button would be fine, although I feel a separate page for downloads would work better than having it on the home page

**Comment by Rexogamer** [comment]
No longer necessary due to the upcoming redesign including a separate download page.

**Pull request closed without merging**